src/Lykke.Service.SmsSender/Sagas/Commands/SendSmsCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Lykke.Service.SmsSender.Core.Domain;
+using ProtoBuf;
+
+namespace Lykke.Service.SmsSender.Sagas.Commands
+{
+    [ProtoContract]
+    public class SendSmsCommand
```

Comment:
we can add requariment here - send ClientId?
will be good in log write info about clientId, becouse if we have problem with client, we can found this event by ClientId

Reply:
the same for the clientId